### PR TITLE
Update collection pages

### LINKED ISSUE #20: Update buy.u.c collection pages
MPT's investigation: https://github.com/CanonicalLtd/Ubuntu-design/issues/161

Recommendations and updated copy docs: https://docs.google.com/document/d/1PI12dk533l4Q3_vW-rpe7bcUpdn9ibp7BCHsxKxSQC4/edit#heading=h.waww95rxd02

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Implemented Vanilla and its patterns on product collection pages

## QA

- Visit https://canonical-staging.myshopify.com/
- In the nav, click any of "Virtual Machines", "Servers" and "Desktops", or click the green "Get started with our Essential plan from $25" CTA in the header strip.
- See that the page you arrive at is not broken, uses Vanilla well, and matches the [proposal](https://docs.google.com/document/d/1PI12dk533l4Q3_vW-rpe7bcUpdn9ibp7BCHsxKxSQC4/edit#)

Fixes #20 & #19

### CODE REVIEW

**Comment by anasereijo** [comment]
@sowasred2012 needs a bit of design love (my fault! we need to plan this in for me) but looks "vanillarised" 👍

**Comment by sowasred2012** [comment]
@anthonydillon this is ready for you to peruse

File: src/snippets/header.liquid [hunk 1]
```diff
@@ -27,47 +27,33 @@
   </header>
 
 {% elsif template contains 'collection' %}
-<div style="display: none;">{{ collection.handle }}</div>
+  <div style="display: none;">{{ collection.handle }}</div>
```
Comment (anthonydillon) [inline src/snippets/header.liquid]: Can you use `u-hide` from vanilla?

**Review by anthonydillon — APPROVED** [review]
All looks good with a suggestion in the code. Also some issues in the HTML (not introduced by this branch) but feel free to catch in a follow up:
- https://validator.w3.org/nu/?doc=https%3A%2F%2Fcanonical-staging.myshopify.com%2Fcollections%2Fubuntu-advantage-for-virtual-guests
- https://validator.w3.org/nu/?doc=https%3A%2F%2Fcanonical-staging.myshopify.com%2F

**Comment by sowasred2012** [comment]
@anthonydillon thanks - good point, I've updated https://github.com/canonical-web-and-design/buy.ubuntu.com/issues/21 with a note about the HTML validation errors

**Pull request merged**